runtime/compiler/x/codegen/J9TreeEvaluator.cpp

```diff
@@ -7293,7 +7314,7 @@ J9::X86::TreeEvaluator::VMnewEvaluator(
    TR_OpaqueClassBlock *clazz      = NULL;
    TR::Register        *classReg   = NULL;
    bool                 isArrayNew = false;
-
+   bool     dynamicArrayAllocation = false;
```

Comment:
Since this isn't used between here and the real initialization below, would you mind moving the declaration down? Right now it looks like something might read this `false`

Reply:
Done